### PR TITLE
Update PowerBI Overview

### PR DESCRIPTION
**Pull request by JeremyLikness**
Updates per feedback from Adam Saxton. Tagging @CamSoper for merge. I will have Power BI team review and provide feedback for sign-off.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/powerbi-embedded.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/powerbi-embedded.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/powerbi-embedded?branch=pr-en-us-274) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C31%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707311232594629-274%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by JeremyLikness** [comment]
Tagging @guyinacube for review.

File: api/overview/azure/powerbi-embedded.md [hunk 1]
```diff
@@ -15,25 +15,64 @@ ms.service: multiple
...
 
+[Power BI](https://powerbi.microsoft.com/) is a cloud-based business analytics service that gives you a single view of your most critical business data.
 
-## Management library
+To learn more about using Power BI with .NET, see [Embedding with Power BI](https://powerbi.microsoft.com/en-us/documentation/powerbi-developer-embedding/).
 
-Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Management.PowerBIEmbedded) directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
+## Client library
+
+Use the client library to connect with PowerBI APIs to access and interact with data sets and reports.
```
Comment (guyinacube) [inline api/overview/azure/powerbi-embedded.md]: Add space for PowerBI - Should be "Power BI"

File: api/overview/azure/powerbi-embedded.md [hunk 2]
```diff
@@ -15,25 +15,64 @@ ms.service: multiple
 
 # Azure PowerBI Embedded libraries for .NET
```
Comment (guyinacube) [inline api/overview/azure/powerbi-embedded.md]: Add space for PowerBI - Should be "Power BI"

File: api/overview/azure/powerbi-embedded.md [hunk 3]
```diff
@@ -15,25 +15,64 @@ ms.service: multiple
...
 ```
 
 > [!div class="nextstepaction"]
-> [Explore the management APIs](/dotnet/api/overview/azure/powerbi/management)
+> [Explore the client APIs](https://powerbi.microsoft.com/documentation/powerbi-developer-rest-api-reference/)
+
+## Samples
 
+* [Power BI Developer Samples](https://github.com/Microsoft/PowerBI-Developer-Samples)
+* [PowerBI .NET GitHub repo](https://github.com/Microsoft/PowerBI-CSharp)
```
Comment (guyinacube) [inline api/overview/azure/powerbi-embedded.md]: Add space for PowerBI - Should be "Power BI" within the brackets [ ]

File: api/overview/azure/powerbi-embedded.md (hunk 2 above)
Comment (guyinacube) [inline api/overview/azure/powerbi-embedded.md]: If we can drop "Azure" from the title, I would prefer that. I think that may be a requirement though. The actual product name is just Power BI Embedded. Not Azure Power BI Embedded.

**Comment by JeremyLikness** [comment]
Made the changes per @guyinacube request. Adam, can you please re-review and use the sign-off hash tag if it looks good? Thanks!

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/powerbi-embedded.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/powerbi-embedded.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/powerbi-embedded?branch=pr-en-us-274) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C3%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708032230162968-274%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request closed without merging**